src/backend/discovery/discovery.lambda.ts

```diff
@@ -357,9 +359,9 @@ function getRelevantVersionInfos(
           }
 
           metrics.putMetric(MetricName.PACKAGE_VERSION_AGE, Date.now() - modified.getTime(), Unit.Milliseconds);
-          const isEligible = usesEligibleLicenses(infos);
+          const isEligible = licenseList.lookup(infos.license ?? 'UNLICENSED') != null;
```

Comment:
nit: personal style choice, feel free to switch or leave as is. (I think even hinting at type coercion is a waste of the reader's time when you can make things explicit).
```suggestion
          const isEligible = licenseList.lookup(infos.license ?? 'UNLICENSED') !== undefined;
```

Reply:
My personal preference is to always use `===` or `!==` **except** when effectively doing "null or undefined".

If you wanted to fix the code so that it was behaving identically, it'd in fact have to be:

```suggestion
          const spdxLicense = licenseList.lookup(infos.license ?? 'UNLICENSED');
          const isEligible = spdxLicense !== null && spdxLicense !== undefined;
```

And quite frankly, I feel that is higher cognitive load than `!= null`.

Reply:
Sure, though it looks like `lookup()` is typed to not return "null". Anyways it doesn't matter too much